lib/browser/tag/tag.js

```diff
@@ -171,10 +173,7 @@ function Tag(impl, conf, innerHTML) {
     // parse layout after init. fn may calculate args for nested custom tags
     parseExpressions(dom, self, expressions)
 
-    // unmount automatically when our parent does
-    if (parent) parent.on('unmount', self.unmount)
```

Comment:
Yeah, a test for this case would be great. I peeked at the code, and have an idea of how to fix it.